frontend/src/metabase-types/api/field.ts

```diff
@@ -118,7 +120,7 @@ export interface Field {
   updated_at: string;
 }
 
-export interface FieldValues {
+export interface FieldValuesResult {
```

Comment:
I renamed this to match the name of the type on the BE, and to differentiate it conceptually from a list of `FieldValue`